crates/rpc-types/src/eth/transaction/mod.rs

```diff
@@ -88,18 +85,11 @@ pub struct Transaction {
     pub max_fee_per_blob_gas: Option<u128>,
     /// Data
     pub input: Bytes,
-    /// All _flattened_ fields of the transaction signature.
-    ///
-    /// Note: this is an option so special transaction types without a signature (e.g. <https://github.com/ethereum-optimism/optimism/blob/0bf643c4147b43cd6f25a759d331ef3a2a61a2a3/specs/deposits.md#the-deposited-transaction-type>) can be supported.
-    #[serde(flatten, skip_serializing_if = "Option::is_none")]
-    pub signature: Option<Signature>,
-    /// The chain id of the transaction, if any.
-    #[serde(
-        default,
-        skip_serializing_if = "Option::is_none",
-        with = "alloy_serde::u64_opt_via_ruint"
-    )]
-    pub chain_id: Option<u64>,
+    /// Transaction signature.
+    pub signature: B1368,
```

Comment:
Don't we need to use any methods on the signature? If so we could use the Signature type here: https://github.com/core-coin/base-rs/blob/main/crates/primitives/src/signature/sig.rs

Reply:
yeah, it make sense. I haven't noticed that we have wrapper for B1368